.github/workflows/pr-realm-js.yml

```
@@ -7,7 +7,7 @@ on:
   workflow_dispatch:
 
 concurrency:
-  group: ${{ github.head_ref || github.run_id }}
+  group: ${{ github.head_ref }}-${{ github.run_id }}
```

@takameyer I believe this was taken from the official docs.

Perhaps we should use the alternative example (using `workflow` instead of `head_ref`) there? https://docs.github.com/en/actions/using-jobs/using-concurrency#example-only-cancel-in-progress-jobs-or-runs-for-the-current-workflow

```suggestion
  group: ${{ github.workflow }}-${{ github.ref }}
```

---

Yeah, I think something is not quite correct here, so I opened this to experiment.  I was noticing that every now and again the newest push would just not run the tests for some reason.  I'll test this out more on Monday

---

This is an important detail from the docs I didn't initially knew nor expected:

> If you have multiple workflows in the same repository, concurrency group names must be unique across workflows to avoid canceling in-progress jobs or runs from other workflows. Otherwise, any previously in-progress or pending job will be canceled, regardless of the workflow.